device/src/main/java/com/aconno/sensorics/device/format/RemoteAdvertisementFormatRepository.kt

```diff
@@ -0,0 +1,105 @@
+package com.aconno.sensorics.device.format
+
+import com.aconno.sensorics.domain.format.AdvertisementFormat
+import com.aconno.sensorics.domain.format.RemoteAdvertisementFormat
+import com.aconno.sensorics.domain.repository.AdvertisementFormatRepository
+import com.aconno.sensorics.domain.repository.FormatRepository
+import io.reactivex.Completable
+import java.text.SimpleDateFormat
+import java.util.*
+
+class RemoteAdvertisementFormatRepository(
```

Comment:
Rename this class

Reply:
Renamed RemoteFormatRepository to keep the same convention as with LocalFormatRepository.